staff/emma-cowdery/e-commerce-protos/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```

Comment:
Yey!!! you shouldn't have that files here. Close that PR and open another one from a new branch with only the right files 🤓